### PR TITLE
Position limits sample file

### PR DESCRIPTION
**Pull request by aliyaladhani**
@Ben-Richards or @Patrick-Willis do you mind taking a quick look please?

### CODE REVIEW

File: Sample-XML/Position Limits.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Snapshot Date="2017-01-03">
+  <Instruments>
+
+	<Future InstrumentID="11" InstrumentName="AGRICULTURE - BURSA MALAYSIA CRUDE PALM OIL GASOIL SPREAD (POG) FUTURES*" Market="CME" DeliveryMonth="2017-03" CommoditySymbol="POG" IsCashSettled="true"/>
+	<Future InstrumentID="12" InstrumentName="AGRICULTURE - BURSA MALAYSIA CRUDE PALM OIL GASOIL SPREAD (POG) FUTURES*" Market="CME" DeliveryMonth="2017-04" CommoditySymbol="POG" IsCashSettled="true"/>
+	
+	<Option InstrumentID="14" InstrumentName="AGRICULTURE - MILK CLASS III MID-SIZE OPTIONS ON FUTURES" Market="CME" DeliveryMonth="2017-02" CommoditySymbol="JQ" IsCashSettled="true" CallOrPut="Call"/>
```
Comment (Ben-Richards) [inline Sample-XML/Position Limits.xml]: Can we make this XCME for Market

File: Sample-XML/Position Limits.xml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Snapshot Date="2017-01-03">
+  <Instruments>
+
+	<Future InstrumentID="11" InstrumentName="AGRICULTURE - BURSA MALAYSIA CRUDE PALM OIL GASOIL SPREAD (POG) FUTURES*" Market="CME" DeliveryMonth="2017-03" CommoditySymbol="POG" IsCashSettled="true"/>
```
Comment (Ben-Richards) [inline Sample-XML/Position Limits.xml]: @Patrick-Willis I may be being stupid here. But do we use IsCashSettled anywhere in the rules atm? Im not aware of seeing anything that distinguishes in the limits between cash and physically settled

Comment (Patrick-Willis) [inline Sample-XML/Position Limits.xml]: No we don't - as long as it's not a needed field for Rapptr in general we can get rid of

Comment (Ben-Richards) [inline Sample-XML/Position Limits.xml]: super - thought so :) Yeah we can get rid of @aliyaladhani no need to add additional stuff we don't need atm

File: Sample-XML/Position Limits.xml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+	<Option InstrumentID="15" InstrumentName="AGRICULTURE - MILK CLASS III MID-SIZE OPTIONS ON FUTURES" Market="XCME" DeliveryMonth="2017-03" CommoditySymbol="JQ" CallOrPut="Call"/>
+	<Option InstrumentID="16" InstrumentName="AGRICULTURE - MILK CLASS III MID-SIZE OPTIONS ON FUTURES" Market="XCME" DeliveryMonth="2017-04" CommoditySymbol="JQ" CallOrPut="Call"/>
+
+  </Instruments>
+
+  <Portfolios>
+    <Portfolio PortfolioId="12">
+
+		<Asset AssetId="11" AssetName="March 17 AGRICULTURE - BURSA MALAYSIA CRUDE PALM OIL GASOIL SPREAD (POG) FUTURES*" InstrumentId="11" Quantity="1500"/>
+		<Asset AssetId="12" AssetName="March 17 AGRICULTURE - BURSA MALAYSIA CRUDE PALM OIL GASOIL SPREAD (POG) FUTURES*" InstrumentId="12" Quantity="-1800"/>
```
Comment (Ben-Richards) [inline Sample-XML/Position Limits.xml]: Should be April 17

**Pull request merged**